packages/cf.js/src/app-instance.ts

```diff
@@ -1,8 +1,28 @@
-import { AppInstanceID, AppInstanceInfo } from "@counterfactual/common-types";
+import {
+  Address,
+  AppABIEncodings,
+  AppInstanceID,
+  AppInstanceInfo,
+  BlockchainAsset
+} from "@counterfactual/common-types";
+import { BigNumber } from "ethers/utils";
 
 export class AppInstance {
   readonly id: AppInstanceID;
-  constructor(readonly info: AppInstanceInfo) {
+  readonly appId: Address;
```

Comment:
So `initialState` is not being set here?

Reply:
So the idea here is to not include the state in an `AppInstance` and call the Node for that AppInstance's state, hence why it's in the proposal params?

Reply:
Good point.

For starters, I don't reckon any dapp would need access to the *initial* state. However, I can imagine wanting immediate access to the latest state, instead of asking for it asynchronously every time you need it (or subscribing to state update events).

@ebryn @joelalejandro thoughts?

Reply:
I believe it makes sense to have the latest state ready as soon as possible.

Reply:
Do you need it on-demand or could it be loaded async? I'm just a bit afraid of race conditions if you rely on a `state` field being up-to-date at all times.

Reply:
@joelalejandro

Reply:
It can be `async` @mvanderh